docs/npm.md

```diff
@@ -1,6 +1,6 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-# npm Install Requirements
+# yarn, npm
```

Comment:
this PR suggests changing this page to an overall doc on installing with yarn, not just the prereqs